### ISSUE TITLE
Option to Change Metabase Background Color Across the App

### ISSUE THREAD

**Issue by ixipixi** [metabase/metabase#41996]
**Is your feature request related to a problem? Please describe.**
In this instance the problem we're trying to solve is differentiating between multiple different internal Metabase instances. 

**Describe the solution you'd like**
To be able to set a background color other than white for the entire Metabase Instance.

**Describe alternatives you've considered**
Different fonts, changing button colors and making other differentiating color palette changes to charts, etc, haven't been sufficient.

**Comment by dhk** [comment]
For those curious about what's driving the request...
we maintain an instance of metabase for our external customers; and another for our internal customers (ie company colleagues)

We have significantly different rules about what a person may build; internally. 

We operate a shared-visibility model, courtesy of metabase's sandboxing feature, which means that we frequently share dashboards with our external customers so that everyone's looking at the same report.

This feature works terrifically well for us. No custom content. 

But, we have this perennial problem where 
1. internal customer: "please build a thing for us"
1. me: right you are, here's the thing. We built it on the internal version of metabase
1. \ <some time passes\>
1. internal customer: "my external customer can't see the thing. The URL 404's" 
1. me: <sad face>sorry. it's internal only for now
1. internal customer: "can't you make it external"
1. me: will take some work
etc, etc. 

yes, serialization and all the rest of it. 

So, we think that if our internal customers see a dashboard that has a, say bright orange pink aspect to it, then they'll be all: Oh. this is internal use only"

Other efforts at education have not been durable.

**Comment by albertoperdomo** [comment]
@ixipixi We recently implemented [custom background illustrations on the landing page](https://www.metabase.com/docs/master/configuring-metabase/appearance#landing-page). While this isn't different background colors, it could actually help with differentiating instances based on their background, maybe?

**Comment by albertoperdomo** [comment]
Thanks for sharing the context around your need @dhk.
Would custom background illustration help?

**Comment by dhk** [comment]
Yes.  Honestly. A collection of  sticks and an army  of proctors is probably  the end state solution

**Comment by RNSS** [comment]
> [@ixipixi](https://github.com/ixipixi) [@albertoperdomo](https://github.com/albertoperdomo) we could change the top bar color via env vars

@paoliniluis, how?
